### PR TITLE
BPSO-59457 - fixed issue where scroll bar doesn't show up until you mousewheel

### PR DESCRIPTION
**Pull request by helrac**
### This project uses [semver](semver.org), please check the scope of this pr:

- [ ] #none# - documentation fixes and/or test additions
- [x] #patch# - backwards-compatible bug fix
- [ ] #minor# - adding functionality in a backwards-compatible manner
- [ ] #major# - incompatible API change

# CHANGELOG
* **Added** mouse event to update perfect scroll (will add/remove scrollbars)

### CODE REVIEW

**Comment by notmessenger** [comment]
@helrac The build is failing

File: tests/integration/components/frost-object-browser-test.js [hunk 1]
```diff
@@ -230,4 +230,31 @@ describe(test.label, function () {
...
+          hook=myHook
+          content=(component 'mock-content' class='mock-content')
+          controls=(component 'mock-controls' class='mock-controls')
+          filters=(component 'mock-filters' class='mock-filters' hook=filterHook)
+        }}
+      `)
+    })
+
+    it('should call the handler displayFilter', function () {
+      $hook(filterHook).css('padding-left', '10px')
```
Comment (notmessenger) [inline tests/integration/components/frost-object-browser-test.js]: This type of setup is supposed to be done in the `beforeEach()` - https://github.com/ciena-frost/ember-frost-test#it-blocks

Comment (helrac) [reply tests/integration/components/frost-object-browser-test.js]: okay, i'll move it

File: tests/integration/components/frost-object-browser-test.js [hunk 2]
```diff
@@ -230,4 +230,30 @@ describe(test.label, function () {
...
+        {{frost-object-browser
+          hook=myHook
+          content=(component 'mock-content' class='mock-content')
+          controls=(component 'mock-controls' class='mock-controls')
+          filters=(component 'mock-filters' class='mock-filters' hook=filterHook)
+        }}
+      `)
+      $hook(filterHook).css('padding-left', '10px')
+      $hook(filterHook).css('padding-bottom', '1500px')
+    })
```
Comment (notmessenger) [inline tests/integration/components/frost-object-browser-test.js]: Should any series of these calls be using `wait()` and `andThen()` calls?  Line 250 and 251 rely on the completion of line 242 and 254 relies on all of them.  But there isn't anything orchestrating this specific order.

**Comment by notmessenger** [comment]
👍

**Pull request merged**